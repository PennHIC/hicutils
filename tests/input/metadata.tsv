replicate_name	subject	biological_sample	disease
HPAP010_rep1	HPAP010	HPAP010	HC
HPAP041_rep1	HPAP041	HPAP041	T1D
