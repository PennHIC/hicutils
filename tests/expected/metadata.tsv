disease	uniques	copies	cdr3_num_nts	avg_v_identity	in_frame	clones	productive_clones
Control	19654	49785	51.37273121476801	0.9696518057362323	0.8786470916186583	16143	14184
T1D	31501	141826	54.0361974625699	0.9620149960916361	0.8677770428717455	16631	14432
